extendr-api/src/wrapper/externalptr.rs

```diff
@@ -3,6 +3,11 @@ use std::any::Any;
 use std::fmt::Debug;
 
 /// Wrapper for creating R objects containing any Rust object.
+/// External pointers (`ExternalPtr`) are accessible from R as an object of class `externalptr`.
+/// Your R code can use external pointers to reference objects in Rust memory.
+///
+/// External pointers can be made avaialable as their underlying type by dereferencing.
+/// In order to deference, the object must either implement the `Copy` trait or be a reference—e.g. `&T`.
```

Comment:
I still don't get this. Doesn't `extptr.deref()` return a `&T`? So it doesn't matter if the `T` implements copy?